### ISSUE TITLE
Crash on attempt to edit SoundPlayer2D with broken dependencies

### ISSUE THREAD

**Issue by syskrank** [godotengine/godot#6661]
**Operating system or device - Godot version:**
linux debian stable ( 8 ) x64; ( note: not using pulseaudio )
godot 2.1 stable official

**Issue description** (what happened, and what was expected):
Godot crashed several times after pressing the 'edit' arrow button on "Samples" property of SoundPlayer2D.

**Steps to reproduce:**
-Create a main level scene
-Create a 'prefab' scene with SoundPlayer2D node in it
-Create new Samples for SoundPlayer2D, save it as a .tres resource
-Import samples, add it to newly created Samples resource
-Make sure that 'prefab' is instanced within level scene
-Save everything, close the project ( exit to the project list )
-Remove samples from directory ( yes, by hand - just shift-delete .smp folks )
-Open project for editing
-Open 'prefab' scene, see the 'broken dependencies' warning
-Try to edit SoundPlayer2D "Samples" property by clicking 'edit' arrow icon.
-Experience the crash.

When Godot engine is launched from the console, I can see the following:

```
ERROR: load_interactive: Condition ' err!=OK ' is true. returned: Ref<ResourceInteractiveLoader>()
   At: core/io/resource_format_binary.cpp:1081.
ERROR: load: Failed loading resource: res://sfx/platform_move.smp
   At: core/io/resource_loader.cpp:218.
ERROR: load_interactive: Condition ' err!=OK ' is true. returned: Ref<ResourceInteractiveLoader>()
   At: core/io/resource_format_binary.cpp:1081.
ERROR: load: Failed loading resource: res://sfx/platform_move.smp
   At: core/io/resource_loader.cpp:218.
WARNING: _parse_ext_resource: Couldn't load external resource: res://sfx/platform_move.smp
   At: scene/resources/scene_format_text.cpp:122.
```

**Comment by Noshyaar** [comment]
First of all thank you for your report and sorry for the delay.

We released Godot 3.0 in January 2018 after 18 months of work, fixing many old issues either directly, or by obsoleting/replacing the features they were referring to.

We still have hundreds of issues whose relevance/reproducibility needs to be checked against the [current stable version](https://godotengine.org/download), and that's where you can help us.
**Could you check if the issue that you described initially is still relevant/reproducible in Godot 3.0 or any newer version, and comment about it here?**

For bug reports, please also make sure that the issue contains detailed steps to reproduce the bug and, if possible, a zipped project that can be used to reproduce it right away. This greatly speeds up debugging and bugfixing tasks for our contributors.

Our Bugsquad will review this issue more in-depth in 15 days, and potentially close it if its relevance could not be confirmed.

Thanks in advance.

*Note: This message is being copy-pasted to many "stale" issues (90+ days without activity). It might happen that it is not meaningful for this specific issue or appears oblivious of the issue's context, if so please comment to notify the Bugsquad about it.*

**Comment by akien-mga** [comment]
As there was no answer to the above request, we're closing this issue as inactive.
If you can still reproduce it, please open a new issue against a currently supported release of Godot.
